src/tools/wasmdump.c

```diff
@@ -139,6 +139,14 @@ int main(int argc, char** argv) {
   // types of information.
 
   s_objdump_options.print_header = 1;
+  if (!s_objdump_options.headers && !s_objdump_options.details && !s_objdump_options.disassemble && !s_objdump_options.raw) {
+    printf("At least one of the following switches must be given:\n");
```

Comment:
Should this output go to stderr? I guess you could also just use several string literals instead of having multiple printf calls.